AEPCampaignClassic/Sources/CampaignClassicState.swift

```diff
@@ -0,0 +1,52 @@
+/*
+ Copyright 2022 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPCore
+import AEPServices
+import Foundation
+
+class CampaignClassicState {
+    private let LOG_TAG = "CampaignClassicState"
+    private(set) var dataStore: NamedCollectionDataStore
+
+    // Privacy status
+    private(set) var privacyStatus: PrivacyStatus = .unknown
+
+    /// Creates a new `CampaignState`.
+    init() {
+        self.dataStore = NamedCollectionDataStore(name: CampaignClassicConstants.DATASTORE_NAME)
+    }
+
+    /// Takes the shared states map and updates the data within the Campaign Classic State.
+    /// - Parameter dataMap: The map containing the shared state data required by the Campaign Extension.
+    func update(dataMap: [String: [String: Any]?]) {
+        for key in dataMap.keys {
```

Comment:
We might want to clean up this code as well as the only shared state needed here is that of Configuration.

Reply:
removed this code.. will redo it with simple steps in the next PR